src/components/Trade/TradeConfirm/ReceiveSummary.tsx

```diff
@@ -50,7 +50,7 @@ export const ReceiveSummary: FC<ReceiveSummaryProps> = ({
 
   const slippageAsPercentageString = bnOrZero(slippage).times(100).toString()
   const amountAfterSlippage = bnOrZero(amount)
-    .times(1 - slippage)
+    .times(1 - Number(slippage))
```

Comment:
`bn(1).minus(slippage)` might be safer

Reply:
I'm all about safety: https://github.com/shapeshift/web/pull/3419/commits/6eac305b76c8cd3cd5ad6cf027309b0c539c1eb2